File: PYTHON/Numeric-Functions/digit_sum.py
def digit_sum (num):
	num = int(num)
	res = int(0)
	while (num > 0):
		temp = int(num % 10)
		res = int(res + temp)
		num = int(num / 10)
	return res

x = int(input("User, enter a number: "))
results = digit_sum(x)
print(str(x) + " --> " + str(results))

File: C/code/digit_sum.c
# include <stdio.h>

int digit_sum (int num) {
	int results = 0;
	while (num > 0) {
		int temp = num % 10;
		results += temp;
		num /= 10;
	}
	return results;
}

int main () {
	int x;
	int results;
	printf("User, enter a number: ");
	scanf("%d", &x);
	results = digit_sum(x);
	printf("%d --> %d\n", x, results);
}
